src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
your tests failed because of this and because we use node version which doesn't support static fields in classes